.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
```

Comment:
this file should not be in pull request diff. It looks like problem with backmerging.